templates/web/base/waste/bulky/summary.html

```diff
@@ -35,8 +35,13 @@ <h2 class="govuk-heading-l govuk-!-margin-bottom-5">Booking Summary</h2>
           </div>
           <div class="govuk-warning-text__content">
               <span class="govuk-warning-text__assistive">Warning</span>
+                [% IF cobrand.moniker == 'sutton' || cobrand.moniker == 'kingston' %]
+                <p class="govuk-!-margin-bottom-0">Please carefully read all the details for your booking including the items as you
+                   won't be able to amend the booking once you have paid for the service.</p>
+                [% ELSE %]
                 <p class="govuk-!-margin-bottom-0">Please read carefully all the details for your booking. You won't be
                   able to amend your details afterwards.</p>
```

Comment:
Hmm, this might not be true for Peterborough soon, once the flag is flipped, will need to remember to tweak this text then.